Framework/Algorithms/inc/MantidAlgorithms/FitPeaks.h

```diff
@@ -310,6 +319,9 @@ class DLLExport FitPeaks : public API::Algorithm {
   //----- Result criterias ---------------
   /// peak positon tolerance case b, c and d
   bool m_peakPosTolCase234;
+
+  /// Supported peak profiles for observation
+  std::vector<std::string> m_observablePeakProfiles;
```

Comment:
Based on your usage of this, it would be better to be a static variable in an anonymous namespace. Also, The values don't need to be sorted.